erl_nif_sys/gen_api.erl

```diff
@@ -33,8 +33,15 @@ version_opts("2.13") -> [{major,2}, {minor,13}, exception, getenv, time,   % erl
 version_opts("2.14") -> [{major,2}, {minor,14}, exception, getenv, time,   % erlang 21.0
                         dirty_scheduler_opt, nif_2_11, nif_2_12, nif_2_13,
                         nif_2_14];
-version_opts(_) ->
-    io:format("Unsupported Erlang version.\n\nIs the erlang_nif-sys version up to date in the Cargo.toml?\nDoes 'cargo update' fix it?\nIf not please report at https://github.com/goertzenator/erlang_nif-sys.\n"),
+version_opts("2.15") -> [{major,2}, {minor,15}, exception, getenv, time,   % erlang 22.0
+                        dirty_scheduler_opt, nif_2_11, nif_2_12, nif_2_13,
+                        nif_2_14, nif_2_15];
+version_opts(Ver) ->
+    io:format(
+        "This OTP release uses the unsupported Erlang NIF version ~p.\n\n"
```

Comment:
`s/the/an`

This OTP release uses an unsupported...

Reply:
Well, the message contains the version, so it reads `This OTP release uses the unsupported Erlang NIF version 2.15` ;)